app/javascript/controllers/setting_controller.js

```diff
@@ -195,6 +193,9 @@ export default class extends Controller {
     const isNightMode = $("body").hasClass("night");
     $("body").toggleClass("night");
     $("#toggle-nightmode").toggleClass("text-primary");
+    var text = $("li a#toggle-nightmode span").text();
```

Comment:
We should not hard code localized messages. Use i18n, also let's make header consistent. Use the same grey color for Day mode/Night mode text.

Reply:
Should we use Nokogiri for this?

Couldn't find a "Rails" way of changing text based on the class of a HTML element

Reply:
@AshNaz87 check out how we've implemented the "copy ayah" feature. When text is copied, we're showing "Done" message. 

https://github.com/quran/quran.com-frontend-v2/blob/master/app/javascript/controllers/verse_controller.js#L64

Reply:
@AshNaz87 can you finish this today? We are going to have production build tonight iA

Reply:
Sure - when do you need this completed by exactly? My timezone is BST